examples/connect_and_publish.py

```diff
@@ -0,0 +1,34 @@
+from streamr.client.client import Client
+
+
+myOption = {'apiKey': 'your-api-key'}
+myClient = Client(myOption)
+
+
+# To connect to the server, you can use connect method
+# This method returns immediately  and you can query the state by isConnected funciton
+myClient.connect()
```

Comment:
Failed to connect - "raise Exception('Already connecting')"

Reply:
This happens due to the autoconnect attribute of the client. We disabled the autoconnect and autodisconnect options.
```
myOption = {'apiKey': 'your-api-key',
            'autoConnect': False, 'autoDisconnect': False}
```